pkg/operator/apis/config/types.go

```diff
@@ -60,6 +60,8 @@ type ControllerConfiguration struct {
 	NetworkPolicy NetworkPolicyControllerConfiguration
 	// VPAEvictionRequirements is the configuration for the VPAEvictionrequirements controller.
 	VPAEvictionRequirements VPAEvictionRequirementsControllerConfiguration
+	// ExtensionVirtualClusterConfig defines the configuration of the GardenExtension controller.
+	ExtensionVirtualClusterConfig ExtensionVirtualClusterConfigControllerConfiguration
```

Comment:
That's a long name 😅 
```suggestion
	GardenExtension GardenExtensionControllerConfiguration
```

Reply:
Well, you need at least one controller for the runtime cluster and potentially a different one for the admissions (or you combine them into one). So you need some other than just `extension` suffix 😞

Reply:
Could we use `GardenExtension` and `GardenAdmission`?

Reply:
Given the names of the controller, we could at least remove the `Config` suffix as for the other controllers.
```
ExtensionVirtualCluster ExtensionVirtualClusterControllerConfiguration
```